harness/harness.rb

```diff
@@ -19,14 +18,28 @@
 
 puts RUBY_DESCRIPTION
 
+if defined?(Process.clock_gettime) && defined?(Process::CLOCK_MONOTONIC)
+  def realtime
+    r0 = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+    yield
+    Process.clock_gettime(Process::CLOCK_MONOTONIC) - r0
+  end
+else
+  require "benchmark"
+
+  def realtime(&block)
+    Benchmark.realtime(&block)
+  end
```

Comment:
While it does work with `miniruby -I/path/to/ruby/lib`, I think it's more useful to use `Time.new` to obviate `-I` in this case.

Reply:
Oh never mind, I was misreading the branches. Since `Process.clock_gettime` does work for miniruby, it seems fine.

Reply:
`optcarrot/tools/shim.rb` has a fallback using `Time.new`, which I think is for very old Rubies, or `gettimeofday` for mruby. I think we don't care about both of them.

Reply:
Good to know, thanks!
https://github.com/Shopify/yjit-bench/pull/309